src/extensions/default/AutoUpdate/main.js

```diff
@@ -0,0 +1,608 @@
+/*
+ * Copyright (c) 2018 - present Adobe Systems Incorporated. All rights reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ *
+ */
+
+define(function (require, exports, module) {
+    "use strict";
+
+    var CommandManager          = brackets.getModule("command/CommandManager"),
+        MainViewManager         = brackets.getModule("view/MainViewManager"),
+        Commands                = brackets.getModule("command/Commands"),
+        AppInit                 = brackets.getModule("utils/AppInit"),
+        UpdateNotification      = brackets.getModule("utils/UpdateNotification"),
+        NodeDomain              = brackets.getModule("utils/NodeDomain"),
+        FileSystem              = brackets.getModule("filesystem/FileSystem"),
+        FileUtils               = brackets.getModule("file/FileUtils"),
+        Strings                 = brackets.getModule("strings"),
+        StateHandler            = require("StateHandler"),
+        DocumentCommandHandlers = brackets.getModule("document/DocumentCommandHandlers"),
+        MessageIds              = require("MessageIds"),
+        UpdateStatus            = require("UpdateStatus"),
+        UpdateInfoBar           = require("UpdateInfoBar");
+
+
+    var _modulePath = FileUtils.getNativeModuleDirectoryPath(module),
```

Comment:
Execute the steps only for `Win` and `Mac`

Reply:
Addressed this in 537430b